zerver/worker/queue_processors.py

```diff
@@ -528,4 +522,4 @@ def consume(self, event):
             bot_handler.handle_message(
                 message=message,
                 bot_handler=self.get_bot_api_client(user_profile),
-                state_handler=self.get_state_handler())
+                state_handler=None)
```

Comment:
Why do we need to pass `None` here, rather than not having the argument?

Reply:
Currently all bots use the interface `handle_message(self, message, bot_handler, state_handler)`. After the discussion with @showell Ithink we are planning to change this, as of now I'm not completely sure what the changes will look like.